src/integrations/ga/GaToSplit.js

```diff
@@ -1,7 +1,257 @@
+  window[gaAlias] = window[gaAlias] || function (...args) {
+    (window[gaAlias].q = window[gaAlias].q || []).push(args);
+  };
+
+  // provides the plugin for use with analytics.js.
+  window[gaAlias]('provide', pluginName, pluginConstructor);
+
+  // @TODO should we registers the plugin on the global gaplugins object?
+  // window.gaplugins = window.gaplugins || {};
+  // window.gaplugins[capitalize(pluginName)] = pluginConstructor;
+}
+
+// Default filter: accepts all hits
+export function defaultFilter() { return true; }
+
+// Default mapping: object used for building the default mapper from hits to Split events
+// @TODO review default mapping.
+const defaultMapping = {
+  eventTypeIdPrefix: {
+    pageview: 'ga-pageview',
+    screenview: 'ga-screenview',
+    event: 'ga-event',
+    social: 'ga-social',
+    timing: 'ga-timing',
+    transaction: 'ga-transaction',
+    item: 'ga-item',
+    exception: 'ga-exception',
+  },
+  eventTypeIdSuffix: {
+    // pageview: 'page',
+    // screenview: 'screenName',
+    event: 'eventAction',
+    social: 'socialAction',
+    // timing: 'timingVar',
+  },
+  eventValue: {
+    event: 'eventValue',
+    timing: 'timingValue',
+  },
+  eventProperties: {
+    pageview: ['page'],
+    screenview: ['screenName'],
+    event: ['eventCategory', 'eventLabel'],
+    social: ['socialNetwork', 'socialTarget'],
+    timing: ['timingCategory', 'timingVar', 'timingLabel'],
+  }
+};
+
+/**
+ * Build a mapper function based on a mapping object
+ *
+ * @param {object} mapping
+ */
+function mapperBuilder(mapping) {
+  return function (model) {
+    const hitType = model.get('hitType');
+
+    let eventTypeId = (mapping.eventTypeIdPrefix[hitType] || 'ga');
+    const suffix = model.get(mapping.eventTypeIdSuffix[hitType]);
+    if (suffix)
+      eventTypeId += '.' + suffix;
+
+    const value = model.get(mapping.eventValue[hitType]);
+
+    const properties = {};
+    const fields = mapping.eventProperties[hitType];
+    if (fields) {
+      for (let i = 0; i < fields.length; i++) {
+        properties[fields[i]] = model.get(fields[i]);
+      }
+    }
+
+    return {
+      eventTypeId,
+      value,
+      properties,
+    };
+  };
+}
+
+export const defaultMapper = mapperBuilder(defaultMapping);
+
+/**
+ * Return a new list of identities removing invalid and duplicated ones.
+ *
+ * @param {Array} identities list of identities
+ * @returns list of valid and unique identities, or undefined if `identities` is not an array.
+ */
+export function validateIdentities(identities) {
+  if (!Array.isArray(identities))
+    return undefined;
+
+  // Remove duplicated identities
+  const uniqueIdentities = uniqO(identities);
+
+  // Filter based on rum-agent identities validator
+  return uniqueIdentities.filter(identity => {
+    if (!identity)
+      return false;
+
+    const maybeKey = identity.key;
+    const maybeTT = identity.trafficType;
+
+    if (!isString(maybeKey) && !isNumber(maybeKey))
```

Comment:
If not yet implemented, add a test to check what happens when you receive numbers.